client/actions/index.js

```diff
@@ -1,4 +1,23 @@
-import request from 'superagent'
+import axios from 'axios'
+
+const ROOT_URL = 'http://localhost:3000/posts'
```

Comment:
I think we can get away with using `'/posts'` as the value for `ROOT_URL`. Currently it won't work when deployed. Be sure and test that it still works after you change it.